.travis.yml

```diff
@@ -10,6 +10,9 @@ matrix:
     - scala: 2.12.8
       env: PLAY_VERSION=2.6
       jdk: openjdk8
+    - scala: 2.12.8
+      env: PLAY_VERSION=2.7
+      jdk: openjdk8
```

Comment:
You can simplify this with (it will get the scala versions from build.sbt):
```
jdk: openjdk8
script: sbt +test
matrix:
  include:
    - env: PLAY_VERSION=2.5
    - env: PLAY_VERSION=2.6
    - env: PLAY_VERSION=2.7
```